### PR TITLE
#154757370 Add a CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by RayNjeri**
#### What does this PR do?
This PR describes Wire's code of conduct and sets guidelines for future contributors.
#### Description of Task to be completed?
Create a CONTRIBUTING.md file
#### What are the relevant pivotal tracker stories?
[#154757370](https://www.pivotaltracker.com/n/projects/2117172)

### CODE REVIEW

**Comment by Mbarak-Mbigo** [comment]
Nice doc, we could add `coding styles/conventions` to adhere to while contributing, you can reference [Airbnb](https://github.com/airbnb/javascript/tree/master/react) as well as the [engineering playbook](https://github.com/andela/engineering-playbook/blob/master/conventions.md) on `Commits, PRs, PT stories`

**Comment by Mercy-Muchai** [comment]
We may need to confirm with technology what Andela products use as contribution guides and what applies here. Will close this for now. Thanks @RayNjeri

**Comment by Mbarak-Mbigo** [comment]
Any reason we don't have reviews on this PR?

File: CONTRIBUTING.MD [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Update the README.md with details of changes to the interface, this includes new environment 
+   variables, exposed ports, useful file locations and container parameters.
+3. Increase the version numbers in any examples files and the README.md to the new version that this
+   Pull Request would represent. 
+4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you 
+   do not have permission to do that, you may request the second reviewer to merge it for you.
+
```
Comment (Mnickii) [inline CONTRIBUTING.MD]: maybe include adherance to a PR template? Is there a PR template?

**Pull request closed without merging**